### PR TITLE
Add project files

### PR DESCRIPTION
**Pull request by amarjin6**
Created 6 classes.

### CODE REVIEW

File: module/main.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import turtle, line, rectangle, polygon, circle, ellipse
+
+
+class Figures:
+    def __init__(self, x, t):
+        self.__x = x
+        if x == 1:
```
Comment (WildTiger404) [inline module/main.py]: That is not a good approach at all. This code violates the open-closed principle
https://stackify.com/solid-design-open-closed-principle/
If I ask you to add new figure you'll have to go through all your code and make changes in all files. That's not good at all

File: module/main.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            self.Rectangle.draw()
+
+        if self.__x == 3:
+            self.Polygon.draw()
+
+        if self.__x == 4:
+            self.Circle.draw()
+
+        if self.__x == 5:
+            self.Ellipse.draw()
```
Comment (WildTiger404) [inline module/main.py]: if self.__x == 6:
if self.__x == 7:
if self.__x == 8:
if self.__x == 9:
....
if self.__x == n:
if self.__x == n+1:
Your opinion. Is it ok?

File: module/main.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+if __name__ == '__main__':
+    win = turtle.Screen()
+    t = turtle.Turtle()
+    print('Welcome to Draw Room!')
+    print(
+        'What to Draw: [1 - Line, 2 - Rectangle, 3 - Polygon(Triangle, Square, Hexagon, N-angle), 4 - Circle, 5 - Ellipse, 0 - Exit]')
+    x = int(input())
+    while x:
+        f = Figures(x, t)
+        f.draw()
```
Comment (WildTiger404) [inline module/main.py]: The task was to draw all figures in one collection

Comment (WildTiger404) [reply module/main.py]: why do we need to input smt?

**Review by WildTiger404 — CHANGES_REQUESTED** [review]

File: module/main.py (hunk 3 above)
Comment (amarjin6) [reply module/main.py]: Ok, I'll draw all figures automatically one by one.

File: module/main.py (hunk 1 above)
Comment (amarjin6) [reply module/main.py]: > That is not a good approach at all. This code violates the open-closed principle https://stackify.com/solid-design-open-closed-principle/ If I ask you to add new figure you'll have to go through all your code and make changes in all files. That's not good at all

Modified module structure according Open-closed principle & added dependency injection approach. Also in main module you can create another class instances to make a set of figures and then draw them together.

File: module/main.py [hunk 4]
```diff
@@ -2,57 +2,52 @@
 
 
 class Figures:
-    def __init__(self, x, t):
-        self.__x = x
-        if x == 1:
-            self.Line = line.Line(int(input('Enter line size:\n')),t)
+    def __init__(self, figure, i, t):
+        w = []
```
Comment (WildTiger404) [inline module/main.py]: who stole your letters? What is this kind of variable namings? It's okay to have i,j iterators but what is W? I don't know this kind of naming conventions

File: module/main.py [hunk 5]
```diff
@@ -2,57 +2,52 @@
...
-            'What to Draw: [1 - Line, 2 - Rectangle, 3 - Polygon(Triangle, Square, Hexagon, N-angle), 4 - Circle, 5 - Ellipse, 0 - Exit]')
-        x = int(input())
-        t.clear()
-        t.setpos(0,0)
-        t.reset()
-
-    t.bye()
+
+    print('[!] Let\'s draw Line')
+    f = Figures(line.Line, 1, t)
```
Comment (WildTiger404) [inline module/main.py]: f, t w, win. Just imagine any other guy looks in your code and trying to figure out smth. That would be not an easy task even for a this simple code

Comment (amarjin6) [reply module/main.py]: Problem only with variable names?

Ok, I will rename them today according to your examples...

Comment (amarjin6) [reply module/main.py]: > f, t w, win. Just imagine any other guy looks in your code and trying to figure out smth. That would be not an easy task even for a this simple code

Variable f - means Figure object, win - screen window, where turtle works.
t - turtle...

Comment (WildTiger404) [reply module/main.py]: yep. My recommendation is to avoid such bad naming practice in future. It's not difficult to name figures instead of f and so on

Comment (amarjin6) [reply module/main.py]: Ok, thanks

**Review by WildTiger404 — APPROVED** [review]

**Pull request merged**